analyses/snv-callers/run_caller_consensus_analysis.sh

```diff
@@ -56,7 +56,8 @@ bedtools intersect \
   -b data/WGS.hg38.mutect2.unpadded.bed > $all_mut_wgs_bed
 
 # Convert GTF to BED file for use in bedtools
-cat data/gencode.v27.primary_assembly.annotation.gtf.gz | gunzip - | grep transcript_id | grep exon | convert2bed --do-not-sort --input=gtf - > $exon_file
+# Here we are only extracting things labeled with CCDS ids, i.e. are coded in protein
+cat data/gencode.v27.primary_assembly.annotation.gtf.gz | gunzip - | grep transcript_id | grep -P "CCDS" | convert2bed --do-not-sort --input=gtf - > $exon_file
```

Comment:
I'm not 100% confident in this, but let's try using structure of the file to make this more specific:

```
gunzip -c data/gencode.v27.primary_assembly.annotation.gtf.gz | awk '$3 ~ /CDS/' | convert2bed --do-not-sort --input=gtf - > $exon_file
```
There is an interesting wrinkle that most exons occur in there twice, but I think when you merge the bed  that should be taken care of.